not enough arguments in call

Hi,
Got error when run main.go:

$ go run main.go
# command-line-arguments
.\main.go:11:32: not enough arguments in call to uploadSession.Upload
        have ()
        want (fbvideo.Option)
.\main.go:11:32: multiple-value uploadSession.Upload() in single-value context

--- a/examples/main.go
+++ b/examples/main.go
@@ -1,14 +1,30 @@
 package main
 
 import (
+	"fmt"
 	"log"
 
 	"github.com/canhlinh/go-fbvideo"
 )
 
 func main() {
-	uploadSession := fbvideo.NewUploadSession("../testdata/video.mp4", 1849649771774166, "EAADGAjjQ7sgBANGPH6KB2fsEZAf7EjCXJR0uoaVxQr25ODc4ZAeyrAoehV3ZAEOz9mrOKD5SxyJbVK7OwgsLeFhoM24c638f91zgNEnQ3ApGnN3OPe3t6S0ZAX3NwlTd3OC0sTZBKbUiw0TOz76l2jbEXwrZArb5qbDwUDfP0ZAsH1fxSv4BE4BRWZCPEzYgvzv8DRfGJgRts2tievuZATy6J")
-	if err := uploadSession.Upload(); err != nil {
+	// You have to export your acess token to FB_ACCESS_TOKEN variable
+	// run command: export FB_ACCESS_TOKEN="Your access token"
+	accessToken := fbvideo.GetAccessToken()
+	me, err := fbvideo.GetMe(accessToken)
+	if err != nil {
+		panic(err)
+	}
+	meID := me["id"].(string)
+
+	uploadSession := fbvideo.NewUploadSession("../testdata/SampleVideo_720x480_10mb.mp4", meID, accessToken)
+
+	// Uploads the video to your wall, only you can see it
+	option := fbvideo.Option{Privacy: &fbvideo.Privacy{Value: fbvideo.PrivacySelf}}
+	videoID, err := uploadSession.Upload(option)
+	if err != nil {
 		log.Fatal(err)
 	}
+
+	fmt.Printf("video_id %s", videoID)
 }

--- a/session.go
+++ b/session.go
@@ -100,7 +100,7 @@ func (uploadSession *UploadSession) Upload(option Option) (string, error) {
 		return "", err
 	}
 
-	uploadSession.fileChunkFolder = "testdata" + "/fb" + session.UploadSessionID
+	uploadSession.fileChunkFolder = os.TempDir() + string(os.PathSeparator) + "fb" + session.UploadSessionID
 	if err := os.MkdirAll(uploadSession.fileChunkFolder, 0777); err != nil {
 		return "", err
 	}

